cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/PostgresConsumerIT.java

```diff
@@ -0,0 +1,141 @@
+
+        // Verify data on Kafka.
+
+        List<String> expected_data_kafka = List.of(
+                "{\"id\":0,\"first_name\":\"first_0\",\"last_name\":\"last_0\",\"days_worked\":23.45}",
+                "{\"id\":1,\"first_name\":\"first_1\",\"last_name\":\"last_1\",\"days_worked\":23.45}",
+                "{\"id\":2,\"first_name\":\"first_2\",\"last_name\":\"last_2\",\"days_worked\":23.45}",
+                "{\"id\":3,\"first_name\":\"first_3\",\"last_name\":\"last_3\",\"days_worked\":23.45}",
+                "{\"id\":4,\"first_name\":\"first_4\",\"last_name\":\"last_4\",\"days_worked\":23.45}");
+
+        Properties props = new Properties();
+        props.put("bootstrap.servers", "10.150.1.22:9092");
+        props.put("group.id", "myapp");
+        props.put("enable.auto.commit", "true");
+        props.put("auto.commit.interval.ms", "1000");
+        props.put("session.timeout.ms", "30000");
+        props.put("key.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
+        // props.put("value.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
+        props.put("value.deserializer", "org.apache.kafka.connect.json.JsonDeserializer");
+
+        KafkaConsumer<String, JsonNode> consumer = new KafkaConsumer<>(props);
+        consumer.subscribe(Arrays.asList("dbserver1.public.test_table"));
+        long expectedtime = System.currentTimeMillis() + 10000;
+        System.out.println("Consumer created");
+        int flag = 0;
+        Thread.sleep(5000);
+        while (System.currentTimeMillis() < expectedtime) {
+            consumer.seekToBeginning(consumer.assignment());
+            ConsumerRecords<String, JsonNode> records = consumer.poll(15);
+            System.out.println("Record count " + records.count());
+            List<String> allLinesKafka = new ArrayList<>();
+            for (ConsumerRecord<String, JsonNode> record : records) {
+                ObjectMapper mapper = new ObjectMapper();
+                if (record.value() != null) {
+                    JsonNode jsonNode = record.value();
+                    String payload = jsonNode.get("payload").toString();
+                    allLinesKafka.add(payload);
+                }
+            }
+            Iterator<String> expected = expected_data_kafka.iterator();
+            int recordsAsserted = 0;
+            for (String line : allLinesKafka) {
+                System.out.println(line);
+                assertEquals(expected.next(), line);
+                ++recordsAsserted;
+                if (recordsAsserted == recordsInserted) {
+                    flag = 1;
+                    break;
+                }
+            }
+            if (flag == 1) {
+                break;
+            }
+        }
+
+        // Verify data on Postgres.
+
+        Class.forName("org.postgresql.Driver");
+        Connection conn = DriverManager.getConnection("jdbc:postgresql://10.150.1.22:5432/postgres", "postgres",
```

Comment:
Instead of hardcoding `10.150.1.22`, you can use `InetAddress..getLocalHost().getHostAddress()` to get the same value.